content/en/docs/reference/glossary/volume.md

```diff
@@ -11,7 +11,7 @@ tags:
 - core-object
 - fundamental
 ---
- A directory containing data, accessible to the {{< glossary_tooltip text="containers" term_id="container" >}} in a {{< glossary_tooltip term_id="pod" >}}.
+ A directory containing data, accessible to the {{< glossary_tooltip text="containers" term_id="container" >}} in a {{< glossary_tooltip text="pod" term_id="pod" >}}.
```

Comment:
The existing page is OK, I think.

To me I'm afraid this feels like a [trivial edit](https://www.kubernetes.dev/docs/guide/pull-requests/#trivial-edits).

If you'd like to make a bigger revision to the glossary entry for _volume_, we'd welcome that @i-dipanshu. I can see that the content is a bit out of date.

Reply:
Thank You,  @sftim for the feedback.
I will update the content glossary entry for volume and more over I'll look into all the glossary entries which are out of date and update them as well.